projects/core/components/group/group.options.ts

```diff
@@ -0,0 +1,23 @@
+import {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk';
+import {TuiOrientation, TuiSizeL} from '@taiga-ui/core/types';
```

Comment:
[Build](https://github.com/taiga-family/taiga-ui/actions/runs/7843809927/job/21404912908?pr=6729) fails:

```ts
Run npx ts-node ./scripts/check-dist.ts
[TASK]: $ grep -niR --color=always 'import("../' /home/runner/work/taiga-ui/taiga-ui/dist
[TASK]: $ grep -niR --color=always 'import(.*/src/.*)' /home/runner/work/taiga-ui/taiga-ui/dist | grep -vE 'demo|.map'
[TASK]: $ grep -niR --color=always 'ɵ0' /home/runner/work/taiga-ui/taiga-ui/dist | grep -vE 'icons/src|demo|.map'
[TASK]: $ grep -niR --color=always 'require(.*/src/.*)' /home/runner/work/taiga-ui/taiga-ui/dist | grep -vE 'taiga-ui-icons-scripts|browser|.map'
Error: Exception [Error]: There are problems with import("../ in:

/home/runner/work/taiga-ui/taiga-ui/dist/core/components/group/group.directive.d.ts:5:    orientation: import("../../../../dist/core/types").TuiOrientation;
/home/runner/work/taiga-ui/taiga-ui/dist/core/components/group/group.directive.d.ts:8:    size: import("../../../../dist/core/types").TuiSizeL;

    at checkIncorrectImports (/home/runner/work/taiga-ui/taiga-ui/scripts/shared/check-incorrect-imports.ts:9:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.all (index 0)
    at async main (/home/runner/work/taiga-ui/taiga-ui/scripts/check-dist.ts:14:9)
Error: Process completed with exit code 1.
```

Probably, this line cause it.